core/templates/dev/head/pages/topic-editor-page/editor-tab/stories-list.directive.html

```diff
@@ -26,7 +26,7 @@
     </td>
     <td>
       <a ng-click="openStoryEditor(story.id)" class="list-summary">
-        <span> <[story.story_publication_status]> </span>
+        <span class="protractor-test-story-publication-status"> <[story.story_publication_status ? "Yes" : "No"]> </span>
```

Comment:
Also, the naming is a bit odd. I would expect a "status" to be published/not-published. What you have here looks more like a boolean, i.e. story.published, story.story_is_published, or similar.

Could you please ensure that the names are consistent throughout this PR? If boolean, use story.published or story.story_is_published for the attribute, and a differently-named variable called new_story_status_is_public when you're modifying that boolean. If you want to keep the "status" name, use string constants rather than true/false -- I think that's what we already do for explorations.

Reply:
Done, though I kept the name of the functions the same (set/changeStoryPublicationStatus).